Production/scripts/ShuffleMergeLaw/analysis/tasks.py

```diff
@@ -0,0 +1,64 @@
+## see https://github.com/riga/law/tree/master/examples/htcondor_at_cern
+
+import six
+import law
+import subprocess
+import os
+import re
+
+from analysis.framework import Task, HTCondorWorkflow
+import luigi
+
+class ShuffleMergeSpectral(Task, HTCondorWorkflow, law.LocalWorkflow):
+  ## '_' will be converted to '-' for the shell command invocation
+  cfg               = luigi.Parameter(description = 'configuration file with the list of input sources')
+  input_path        = luigi.Parameter(description = 'input path with tuples for all the samples')
+  output_path       = luigi.Parameter(description = 'output directory')
+  pt_bins           = luigi.Parameter(description = 'pt bins')
+  eta_bins          = luigi.Parameter(description = 'eta bins')
+  mode              = luigi.Parameter(description = 'merging mode: MergeAll or MergePerEntry')
+  input_spec        = luigi.Parameter(description = '')
+  tau_ratio         = luigi.Parameter(description = '')
+  disabled_branches = luigi.Parameter(description = 'disabled-branches list of branches to disabled in the input tuples',
+                                      default = '')
+  n_threads         = luigi.IntParameter(description = 'number of threads', default = 1)
+  exp_disbalance    = luigi.IntParameter(description = '')
+  n_jobs            = luigi.IntParameter(description = 'number of HTCondor jobs to run')
+  start_entry       = luigi.FloatParameter(description = 'starting point')
+  end_entry         = luigi.FloatParameter(description = 'ending point')
+  
```

Comment:
there will be also a parity-related argument. Please, sync modifications with Mykyta.

Reply:
all the S&M arguments are now in the corresponding law task
https://github.com/lguzzi/TauMLTools/blob/law_tool/Analysis/law/ShuffleMergeSpectral/tasks.py#L16-L35